ReactiveCocoa/Swift/Observer.swift

```diff
@@ -55,3 +55,21 @@ public struct Observer<Value, Error: ErrorType> {
 		action(.Interrupted)
 	}
 }
+
+/// A protocol for type-constrained extensions of `Observer`.
+public protocol ObserverType {
```

Comment:
Sure thing, just copied the ordering from `Signal.swift` and `SignalProducer.swift`.